etl/steps/data/garden/artificial_intelligence/2024-04-02/dynabench.meta.yml

```diff
@@ -0,0 +1,40 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    processing_level: minor
+    description_processing: >
+      We mapped the benchmarks to their respective domains based on a review of each benchmark's primary focus and the specific capabilities it tests within AI systems:
```

Comment:
Oh sorry it's this bit that hasn't formatted nicely 
![Uploading Screenshot 2024-04-02 at 15.18.05.png…]()

Reply:
Maybe you need a |- instead of >

Reply:
Yeh you just need `description_processing: |-` instead

Reply:
ah yeaa sorry was in a bit of a rush with this one and missed the formatting :) thanks a lot for spotting it though!